README.md

```diff
@@ -0,0 +1,7 @@
+# obs_subaru
+
+Subaru-specific (hsc,  hscSim, suprimecam) configuration and tasks for the LSST Data Management Stack
```

Comment:
`hscSim` is long dead.

Reply:
I lifted that description from the package description on the main page... I'll fix it here, but someone (admin?) needs to fix it on the main github page.